setup/management.sh

```diff
@@ -61,6 +61,41 @@ if [ ! -f $STORAGE_ROOT/backup/secret_key.txt ]; then
 	$(umask 077; openssl rand -base64 2048 > $STORAGE_ROOT/backup/secret_key.txt)
 fi
 
+
+# Download jQuery and Bootstrap local files
+echo "Installing Mail-in-a-Box system management web assets..."
+
+# jQuery CDN URL
+jquery_version=2.1.4
```

Comment:
Because each file has a hash listed below, it's better to keep all of the version-specific values together. So the version numbers and urls should be moved to be right above the wget_verify lines.